docs/content/docs/build/containers.md

```diff
@@ -19,54 +19,75 @@ In the below scenarios, `client` represents the service-specific client. See the
 
 ### Container schema
 
-You must define a schema that represents the structure of the data within your container. A schema can include `initialObjects` that are always available and types that can be dynamically created by the container at runtime. The same schema definition must be provided for creation and subsequent loading of the container. For more information on `initialObjects` and dynamic object creation see [Data modeling](./data-modeling.md).
+You must define a schema that represents the structure of the data within your container. A schema can include `initialObjects` that are always available and types that can be dynamically created by the container at runtime. The same schema definition must be provided for creation and subsequent loading of the container. For more information on `initialObjects` and dynamic object creation see
+[Data modeling]({{< relref "data-modeling.md" >}}).
 
 This example schema defines two initial objects, and declares `SharedCell` and `SharedString` as shared object types that can be dynamically created at runtime.
 
-```typescript
-const schema = {
-    name: "example-container",
+```typescript {linenos=inline}
+const containerSchema = {
     initialObjects: {
-        layout: SharedDirectory,
+        layout: SharedMap,
         text: SharedString
     },
     dynamicObjectTypes: [ SharedCell, SharedString ],
-}
+};
 ```
 
 ### Creating a container
 
 Containers are created from the service-specific client's `createContainer` function. You must provide a config that is specific to the service and a schema object that defines the container schema.
 
-```typescript {hl_lines=[10]}
+```typescript {linenos=inline,hl_lines=[7,8]}
 const schema = {
-    name: "example-container",
     initialObjects: {
         layout: SharedMap,
     },
-}
-const { container, containerServices} =
+};
+
+const { container, containerServices } =
```

Comment:
```suggestion
const { container, services} =
```

Reply:
Aren't we calling these container services?

Reply:
https://github.com/microsoft/FluidFramework/blob/600e75d4f845c383b2667d2d0ba320d176864e4b/packages/framework/tinylicious-client/src/TinyliciousClient.ts#L75

Reply:
changed 2 weeks ago via https://github.com/microsoft/FluidFramework/pull/7254

Reply:
I see that, but if we are calling these things "container services" then I don't see the value in changing the variable name.

Reply:
it's not a tuple return, `services` is the value returned via destructuring. `containerServices` will come back undefined/throw a type error

Reply:
Ahhhhh... I am once again thrown off by destructuring.